go/cmd/vtctldclient/command/onlineddl.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2023 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package command
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"vitess.io/vitess/go/cmd/vtctldclient/cli"
+	"vitess.io/vitess/go/protoutil"
+	"vitess.io/vitess/go/vt/schema"
+	"vitess.io/vitess/go/vt/vtctl/schematools"
+
+	vtctldatapb "vitess.io/vitess/go/vt/proto/vtctldata"
+)
+
+var (
+	OnlineDDL = &cobra.Command{
+		Use:                   "OnlineDDL <cmd> <keyspace> [args]",
```

Comment:
unfortunately cobra won't support a positional argument before the subcommand name (or at least i couldn't see an easy/obvious way)